server/src/server_wrapper.cc

```diff
@@ -32,6 +32,7 @@
 #include <string>
 #include <thread>
 #include <vector>
+#include "triton/developer_tools/server_wrapper.h"
```

Comment:
Duplicated as `triton/developer_tools/server_wrapper.h` is already included at line #27.